freecad/gears/connector.py

```diff
@@ -49,19 +49,63 @@ def __init__(self, obj, master_gear, slave_gear):
         obj.addProperty("App::PropertyString", "version", "version", "freecad.gears-version", 1)
         obj.addProperty("App::PropertyLink","master_gear","gear","master gear", 1)
         obj.addProperty("App::PropertyLink","slave_gear","gear","slave gear", 1)
+        obj.addProperty("App::PropertyLink","master_gear_connector_parent","gear","master gear parent object for connector animation", 1)
+        obj.addProperty("App::PropertyLink","slave_gear_connector_parent","gear","slave gear parent object for connector animation", 1)
         obj.addProperty("App::PropertyAngle", "angle1", "gear", "angle at which second gear is placed", 0)
         obj.addProperty("App::PropertyAngle", "angle2", "gear", "angle at which second gear is placed", 1)
         obj.version = __version__
         obj.master_gear = master_gear
         obj.slave_gear = slave_gear
+        obj.master_gear_connector_parent = master_gear
+        obj.slave_gear_connector_parent = slave_gear
```

Comment:
we have to pick the right naming convention now, rather than later... as it's saved to file